package.json

```diff
@@ -1,7 +1,7 @@
 {
   "type": "module",
   "name": "@playkostudios/wle-cleaner",
-  "version": "0.9.0",
```

Comment:
this is not a backwards compatible update:
- wonderland needs to be upgraded
- the default behaviour for throwing exceptions was changed

for updates like these you would change the minor (or major, if this wasn't a zero-versioned project)